extensions/amp-story-interactive/amp-story-interactive.md

```diff
@@ -1,10 +1,308 @@
+
+[sourcecode:html]
+<amp-story-interactive-poll
+    id="season-poll"
+    theme="dark"
+    endpoint="https://endpoint.com/v1/interactives"
+    prompt-text="Pick a season"
+    option-1-text="Spring" option-1-confetti="🌼"
+    option-2-text="Summer" option-2-confetti="☀️"
+    option-3-text="Fall" option-3-confetti="🍁"
+    option-4-text="Winter" option-4-confetti="☃️">
+</amp-story-interactive-poll>
+[/sourcecode]
+
+### amp-story-interactive-quiz
+
+The `amp-story-interactive-quiz` element provides a guessing experience with 2-4 options, one of which is correct. It displays the voting percentages after the user makes a selection. The user selection is green if correct and red if incorrect.
+
+Display different categories based on percentage of correct user answers by pairing `amp-story-interactive-quiz` with `amp-story-interactive-results`. Add a prompt for extra context.
+
+<amp-img src="https://github.com/mszylkowski/amphtml/raw/interactive_docs/extensions/amp-story-interactive/img/quiz-raw.png" layout="intrinsic" width="400" height="450">
+
+[sourcecode:html]
+<amp-story-interactive-quiz
+    id="arts-quiz"
+    endpoint="https://endpoint.com/v1/interactives"
+    prompt-text='Who was the artist that created the famous painting "The Last Supper"?'
+    option-1-text="Michelangelo"
+    option-2-text="Leonardo da Vinci" option-2-correct option-2-confetti="🎨"
+    option-3-text="Rahael"
+    option-4-text="Sandro Boticelli">
+</amp-story-interactive-quiz>
+[/sourcecode]
+
+### amp-story-interactive-results
+
+The `amp-story-interactive-result` element displays a customized state defined by the user's selection from previous polls or quizzes. This element requires use of a poll or quiz from previous pages to calculate an answer-based state. Each result state uses a category that may include an image, title and description to display to the user.
+
+<amp-img src="https://github.com/mszylkowski/amphtml/raw/interactive_docs/extensions/amp-story-interactive/img/results-raw.png" layout="intrinsic" width="400" height="500">
+
+[sourcecode:html]
+<amp-story-interactive-results
+    theme="dark"
+    prompt-text="You are a"
+    option-1-text="Dog" option-1-image="dog.png"
+    option-1-description="You always have energy and love being with friends. Outdoors is your favorite place"
+    option-2-text="Cat" option-2-image="cat.png"
+    option-2-description="Cats are great animals for WFH">
+</amp-story-interactive-results>
+[/sourcecode]
+
+## Attributes
+
+The interactive experience elements from `amp-story-interactive` share an API language for customizing options.
+
+### id (required for binary-poll, poll, quiz)
+
+Element ID that identifies the interactive component in the story, and is used to compose the [`interactiveId`](#interactiveId) that is sent to the backend. Should be unique for each component in the story.
+
+### endpoint (required for binary-poll, poll, quiz)
+
+Complete URL of backend. Stores interactive element votes.
```

Comment:
Let's add something about the default Google hosted backend?

Reply:
Even though we already have the other section on the backends?

Reply:
Google will have a backend that conforms to the API described in `Aggregate data source` that anyone will be able to plug and use right away